# Housing Market

### Introduction:

This time we will create our own dataset with fictional numbers to describe a house market. As we are going to create random data don't try to reason of the numbers.

### Step 1. Import the necessary libraries

In [1]:
import pandas as pd
import numpy as np

### Step 2. Create 3 differents Series, each of length 100, as follows: 
1. The first a random number from 1 to 4 
2. The second a random number from 1 to 3
3. The third a random number from 10,000 to 30,000

In [18]:
arr1 = np.random.randint(1,5,100)
arr2 = np.random.randint(1,4,100)
arr3 = np.random.randint(10000,30001,100)

s1, s2, s3 = pd.Series(arr1), pd.Series(arr2), pd.Series(arr3)

### Step 3. Let's create a DataFrame by joinning the Series by column

In [19]:
df = pd.concat([s1, s2, s3], axis=1)
df

0  1      2
0   4  1  14498
1   1  2  14173
2   4  2  15700
3   1  2  27682
4   1  1  25606
.. .. ..    ...
95  3  1  21735
96  1  3  24370
97  2  3  15147
98  4  1  14795
99  1  1  20705

[100 rows x 3 columns]

### Step 4. Change the name of the columns to bedrs, bathrs, price_sqr_meter

In [20]:
df.columns = ['bedrs', 'bathrs', 'price_sqr_meter']
df

bedrs  bathrs  price_sqr_meter
0       4       1            14498
1       1       2            14173
2       4       2            15700
3       1       2            27682
4       1       1            25606
..    ...     ...              ...
95      3       1            21735
96      1       3            24370
97      2       3            15147
98      4       1            14795
99      1       1            20705

[100 rows x 3 columns]

### Step 5. Create a one column DataFrame with the values of the 3 Series and assign it to 'bigcolumn'

In [23]:
new_df = pd.DataFrame()
new_df['big_column'] = pd.concat([s1, s2, s3]).reset_index(drop=True)
new_df

big_column
0             4
1             1
2             4
3             1
4             1
..          ...
295       21735
296       24370
297       15147
298       14795
299       20705

[300 rows x 1 columns]

### Step 6. Oops, it seems it is going only until index 99. Is it true?

### Step 7. Reindex the DataFrame so it goes from 0 to 299